sdk/storage/azure-storage-files-shares/inc/azure/storage/files/shares/protocol/share_rest_client.hpp

```diff
@@ -1898,9 +1889,9 @@ namespace Azure { namespace Storage { namespace Files { namespace Shares {
             {
               path.emplace_back(XmlTagName::c_StorageServiceProperties);
             }
-            else if (std::strcmp(node.Name, "Protocol") == 0)
+            else if (std::strcmp(node.Name, "ProtocolSettings") == 0)
```

Comment:
What motivated this rename?

Reply:
A failed unit test. The unit test is disabled due to service not ready but failed in a test environment that has this feature enabled.